tests/test_project.py

```diff
@@ -10,6 +10,7 @@
     'path_with_namespace': 'cool/project',
     'ssh_url_to_repo': 'ssh://blah.com/cool/project.git',
     'merge_requests_enabled': True,
+    'default_branch': 'master',
```

Comment:
Can't we use here the env variable `CI_DEFAULT_BRANCH` which is already provided by gitlab?

Reply:
It's provided both by the env var and by [the API](https://docs.gitlab.com/ee/api/projects.html#get-single-project) so potentially we could do either, but the existing code seems to prefer the API over env vars, so I stuck with that